content/protection.md

```diff
@@ -1,18 +1,33 @@
 ---
-description: Compensation you can claim if your provider can’t pay you.
+description: Find out what compensation you get if your provider can’t pay your pension or annuity.
 ---
 # How your pension is protected
 
-If your provider is authorised by the Financial Conduct Authority (FCA) and the Prudential Regulation Authority (PRA) you can claim compensation for non-payments from the [Financial Services Compensation Scheme](http://www.fscs.org.uk/) (FSCS).
+How your pension is protected depends on your pension type.
 
-The limits for compensation are:
+## Defined contribution pensions
 
-- personal pensions with an authorised insurance company – 100% of your pot’s value
-- most defined contribution workplace pensions with an authorised insurance company – 100% of your pot’s value
-- Self-Invested Personal Pensions (SIPPS) – different compensation limits depending on the type of SIPP – talk to your provider to find out what compensation cover you have
+If your employer goes out of business you won't lose your pension pot. This is because [defined contribution](/pension-types) pensions are usually run by pension providers, not employers.
 
-Once you’ve taken money out of your pot you can claim for:
+^Some defined contribution pensions are run by a trust chosen by the employer. Contact your employer for exact compensation details.^
 
-- money in an account with a bank, building society or credit union – up to £75,000
-- investments – up to £50,000
-- annuities with an authorised insurance company – 100% of your pot’s value
+## If your provider can't pay
```

Comment:
There are a bunch of regular quotes you've introduced in this diff - `'`, could we change them all to smart quotes - `’`? 

Give me a shout if this isn't clear, or you need help 👋